[v10.0.0-rc.1] Breaks PageMeta augmentation

### Environment

Reproduction env:
- Operating System: Linux
- Node Version:     v20.19.1
- Nuxt Version:     3.17.5
- CLI Version:      3.25.1
- Nitro Version:    2.11.12
- Package Manager:  pnpm@8.15.6
- Builder:          -
- User Config:      modules, future, compatibilityDate, devtools, i18n
- Runtime Modules:  @nuxtjs/i18n@10.0.0-rc.1
- Build Modules:    -

Local dev env:
- Operating System: Linux
- Node Version:     v22.16.0
- Nuxt Version:     3.17.5
- CLI Version:      3.25.1
- Nitro Version:    2.11.13
- Package Manager:  pnpm@10.12.4
- Builder:          -
- User Config:      modules, future, compatibilityDate, devtools, experimental, plugins, css, app, nitro, router, routeRules, runtimeConfig, fonts, i18n, icon, image, pwa, robots, security, typescript, vite, webVitals
- Runtime Modules:  nuxt-security@2.2.0, @pinia/nuxt@0.11.1, @nuxtjs/i18n@10.0.0-rc.1, @nuxtjs/robots@5.2.11, @nuxt/content@3.6.1, @nuxt/fonts@0.11.4, @nuxt/icon@1.15.0, @nuxt/image@1.10.0, @nuxtjs/color-mode@3.5.2, @vueuse/nuxt@13.4.0, @vueuse/motion/nuxt@3.0.3, @vite-pwa/nuxt@1.0.4, @nuxtjs/web-vitals@0.2.7, @nuxt/eslint@1.4.1, @nuxt/test-utils/module@3.19.1
- Build Modules:    -

### Reproduction

https://stackblitz.com/edit/github-jjpn8kps

Run `pnpm typecheck` after installing dependencies (`pnpm install`), and see the error.

### Describe the bug

The next i18n implementation breaks types augmentation, for example, defining PageMeta additional props leads to TS error:
// auth.d.ts
```ts
type AccountRoleType = 'admin' | 'user'

declare module '#app' {
  interface PageMeta {
    roles?: AccountRoleType[]
  }
}

export {}
```
and in middleware/auth.ts
```ts
export default defineNuxtRouteMiddleware(async (to) => {
  if (!to.meta.roles?.includes('admin')) {
    throw createError({
      statusCode: 404,
      message: 'Not found',
      fatal: true,
    })
  }
})
```

leads to this TS error:
```
error TS2339: Property 'includes' does not exist on type '{}'.
```

### Additional context

If I remember correctly, nuxt/i18n module v10.0.0-beta.1 was working correctly. A couple of days ago I tried beta.10 and today - rc.1, and such TS error start throwing.

### Logs

```shell
pnpm typecheck

> nuxt-app@ typecheck /home/projects/github-jjpn8kps
> nuxi typecheck


[2:28:51 PM]  ERROR  (node:19) ExperimentalWarning: WASI is an experimental feature and might change at any time
(Use node --trace-warnings ... to show where the warning was created)


 WARN  Slow module @nuxt/devtools took 23273.91ms to setup.                           2:29:30 PM

ℹ Running with compatibility version 4                                          nuxt 2:29:41 PM
app/middleware/auth.ts:2:23 - error TS2339: Property 'includes' does not exist on type '{}'.

2   if (!to.meta.roles?.includes('admin')) {
                        ~~~~~~~~


Found 1 error in app/middleware/auth.ts:2


 ERROR  Process exited with non-zero status (2)                                       2:30:30 PM

    at new x (node_modules/.pnpm/tinyexec@1.0.1/node_modules/tinyexec/dist/main.js:440:5)
    at R._waitForOutput (node_modules/.pnpm/tinyexec@1.0.1/node_modules/tinyexec/dist/main.js:548:13)
    at async Object.run (node_modules/.pnpm/@nuxt+cli@3.25.1/node_modules/@nuxt/cli/dist/chunks/typecheck.mjs:60:7)
    at async runCommand (node_modules/.pnpm/citty@0.1.6/node_modules/citty/dist/index.mjs:337:16)
    at async runCommand (node_modules/.pnpm/citty@0.1.6/node_modules/citty/dist/index.mjs:328:11)
    at async Module.runMain (node_modules/.pnpm/citty@0.1.6/node_modules/citty/dist/index.mjs:466:7) 



 ERROR  Process exited with non-zero status (2)                                       2:30:30 PM

 ELIFECYCLE  Command failed with exit code 1.
```

--- a/src/gen.ts
+++ b/src/gen.ts
@@ -2,7 +2,6 @@ import { isString } from '@intlify/shared'
 import { genDynamicImport, genSafeVariableName, genString } from 'knitwork'
 import { resolve, relative, join, basename } from 'pathe'
 import { asI18nVirtual } from './transform/utils'
-import { resolveModule } from '@nuxt/kit'
 
 import type { Nuxt } from '@nuxt/schema'
 import type { LocaleObject } from './types'
@@ -214,11 +213,6 @@ declare module '#app' {
   interface PageMeta extends I18nMeta {}
 }
 
-
-// NOTE: this is a workaround for Nuxt <3.16.2
-declare module '${resolve(resolveModule('nuxt'), '../pages/runtime/composables')}' {
-  interface PageMeta extends I18nMeta {}
-}
 
 declare module 'vue-router' {
   interface RouteMeta extends I18nMeta {}

